docs/CHANGELOG.rst

```diff
@@ -22,6 +22,8 @@ Added
 Changed
 -------
 - Refactor linters, check PEP 8 and PEP 257
+- Split expression engines into expression engines and execution engines
+- Use Jinja2 instead of Django Template syntax
```

Comment:
No, this is because we will require some Jinja2 API to implement inline expressions correctly in workflows. And it also reduces coupling between processes and Django, which is IMO a good thing.